generated/installer/ibm-spectrum-scale-csi-operator-ocp-rhel.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     product: ibm-spectrum-scale-csi
     release: ibm-spectrum-scale-csi-operator
   annotations:
-    productVersion: 2.13.0
+    productVersion: 2.14.0
 spec:
   replicas: 1
```

Comment:
`generated/installer/ibm-spectrum-scale-csi-operator-ocp-rhel.yaml` is not required from CSI 2.13.0 , so we should remove this file

Reply:
@saurabhwani5 thanks! I deleted the file 😄